content/api/commands/attachfile.md

```diff
@@ -0,0 +1,213 @@
+---
+title: attachFile
+---
+
+Attaches files to HTML5 input element, or simulated dragging files onto the browser.
+
+## Syntax
+
+```javascript
```

Comment:
This could be confusing since the method must be chained. I thinks it's ok to just indicate each as `.attachFile(foo)`, like we do in the `.within()` page.

Reply:
Good call, done.